bullet_train-fields/app/javascript/controllers/fields/color_picker_controller.js

```diff
@@ -146,7 +141,12 @@ export default class extends Controller {
     this.pickr.destroy()
   }
 
+  get colorOptions () {
+    const colorButtons = this.colorOptionsTarget.querySelectorAll('button[data-color]')
+    return Array.prototype.slice.call(colorButtons).map((el) => { return el.dataset.color })
```

Comment:
I think we can use `Array.from` here, which is a little simpler. I can't remember if `...` is required though.

We can also cut `return` if we don't use wrapping braces in the anonymous function.

```suggestion
    return Array.from(...colorButtons).map((el) => el.dataset.color)
```